.github/workflows/ci-core-reusable.yml

```diff
@@ -202,7 +202,7 @@ jobs:
       # `sleep 5` because we need to wait until server started properly
       - name: Run server
         run: |
-          ci_run zk server --components=$SERVER_COMPONENTS &>server.log &
+          ci_run zk server --use-node-framework --components=$SERVER_COMPONENTS &>server.log &
```

Comment:
This means that for each workflow in the matrix,
1) we will run integration tests with the server initialized via framework, but
2) when we run EN integration tests, the server will be initialized the old way.

So, we will cover all the feature sets without having any more workflows.